renovate.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
A bit nervous based on the PR description so creating a comment to prevent accidental merging

Reply:
I converted the PR to draft while I tweak the configuration.